utils/eventcharges.go

```diff
@@ -364,10 +380,11 @@ type ExtEventCharges struct {
 
 	Charges []*ChargeEntry
 
-	Accounting  map[string]*ExtAccountCharge
-	UnitFactors map[string]*ExtUnitFactor
-	Rating      map[string]*ExtRateSInterval
-	Accounts    map[string]*ExtAccount
+	Accounting    map[string]*ExtAccountCharge
+	UnitFactors   map[string]*ExtUnitFactor
+	Rating        map[string]*ExtRateSInterval
+	IntervalRates map[string]*ExtIntervalRate
```

Comment:
Please rename to Rates instead of IntervalRates, so we can be in sync with RateProfileCost.